deployment/deploy-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: deploy-config
+data:
+  STORAGE_SYSTEM: cloud_spanner
+  STORAGE_FLAG: --cloudspanner_uri=projects/PROJECT_ID/instances/trillian-spanner/databases/trillian-db
+  SIGNER_DEQUEUE_BUCKET_FRACTION: "0.0078"
+  SIGNER_BATCH_SIZE: "1800"
+  SIGNER_INTERVAL: "2ms"
+  SIGNER_NUM_SEQUENCERS: "10"
+  SIGNER_MASTER_HOLD_JITTER: "7200s"
```

Comment:
These SIGNER_* fields should include the flag names, e.g. https://github.com/google/trillian/blob/a6efc69d95ffe1465f7efa256d2fd294de588787/examples/deployment/kubernetes/trillian-ci-spanner.yaml#L16-L20